pages/a/1k1-azure-post.html

```diff
@@ -0,0 +1,239 @@
+title: Crate guest post for Azure - Setting up Azure for Crate
+author: Chris Ward
+description: Setting up a 1001 node Crate cluster on Azure with Docker Swarm
+created: 2016-03-29
+status: publish
+post_type: post
+tags: azure, arm
+category: developernews
+
+# Crate guest post for Azure - Setting up Azure for Crate
```

Comment:
the headline could reflect the content of the post better!
also: I would not be interested in reading this post because of this headline - needs to be more catchy!